build.gradle

```diff
@@ -60,33 +60,18 @@ allprojects {
                 username geArtifactoryReader
                 password geArtifactoryReaderPassword
             }
-            url 'https://devcloud.swcoe.ge.com/artifactory/PREDIX'
-        }
-        maven {
-            credentials {
-                username geArtifactoryReader
-                password geArtifactoryReaderPassword
-            }
-            url 'https://devcloud.swcoe.ge.com/artifactory/PREDIX-SNAPSHOT'
+            url 'https://artifactory.build.ge.com/predix-virtual'
         }
         // Repositories required to download jwt-bearer-key-provider
         maven {
             credentials {
                 username geArtifactoryReader
                 password geArtifactoryReaderPassword
             }
-            url 'https://devcloud.swcoe.ge.com/artifactory/MAAXA-MVN'
-        }
-        maven {
-            credentials {
-                username geArtifactoryReader
-                password geArtifactoryReaderPassword
-            }
-            url 'https://devcloud.swcoe.ge.com/artifactory/MAAXA-MVN-SNAPSHOT'
-        }
-        maven {
-            url 'https://build.shibboleth.net/nexus/content/repositories/releases/'
+            url 'https://artifactory.build.ge.com/artifactory/MAAXA'
```

Comment:
MAAXA repo is not required now.  jwt-bearer-key-provider is available in predix-virtual now.  We could remove lines 65-72.

Reply:
This was fixed with the last commit